localstack/services/events/models.py

```diff
@@ -65,6 +106,32 @@ def __post_init__(self):
 RuleDict = dict[RuleName, Rule]
 
 
+@dataclass
+class Archive:
+    name: ArchiveName
+    region: str
+    account_id: str
+    event_source_arn: Arn
+    description: ArchiveDescription = None
+    event_pattern: EventPattern = None
+    retention_days: RetentionDays = None
+    state: ArchiveState = ArchiveState.DISABLED
+    arn: Arn = field(init=False)
+    creation_time: Timestamp = None
+    size_bytes: int = 0  # TODO how to deal with updating this value?
+    events: FormattedEventDict = field(default_factory=dict)
+
+    def __post_init__(self):
+        self.arn = f"arn:aws:events:{self.region}:{self.account_id}:archive/{self.name}"
```

Comment:
just stumbled across the PR and dropping a random comment: a more common pattern would be

```python
@property
def arn(self) -> str:
    return f"..."
```

and also we should be using the arn utils for this :-)

Reply:
fixed in: https://github.com/localstack/localstack/pull/10890/commits/b288f1358589000947ae56274acc58d26cfe5e11